SPECS/asciidoc/asciidoc.spec

```diff
@@ -1,6 +1,6 @@
 Summary:        AsciiDoc is a human readable text document format
 Name:           asciidoc
-Version:        9.1.0
+Version:        10.2.0
 Release:        1%{?dist}
 License:        GPLv2
```

Comment:
https://github.com/asciidoctor/asciidoc-docs/blob/main/README.adoc says some different license, not GPLv2.

Reply:
@ameyapu, Thank you for the review. The repo that we are using is given below. 
https://github.com/asciidoc-py/asciidoc-py/blob/main/LICENSE
According to this It seems the license is GPLv2  only.  I think we are good here. Let me know If I am missing anything.